enhancements/20190919-OVN-Kubernetes.md

```diff
@@ -107,6 +118,12 @@ version skew can occur between nodes.
 
 ## Drawbacks
 
+OVN currently has no way to implement:
+- Session affinity for Services
+- "except" blocks for NetworkPolicy
```

Comment:
Neither does openshift-sdn...

Reply:
It sounds like adding this clarification would be helpful